main.go

```diff
@@ -91,6 +92,8 @@ func main() {
 	logger.FatalOnError(envconfig.Process("submariner", &submSpec), "Error processing env vars")
 
 	logger.Infof("Parsed env variables: %#v", submSpec)
+	logger.Infof("Proxy env variables: HTTP_PROXY: %v, HTTPS_PROXY: %v, NO_PROXY: %v", os.Getenv("HTTP_PROXY"),
```

Comment:
`ProxyFromEnvironment` also checks the lowercase variants, see https://cs.opensource.google/go/x/net/+/refs/tags/v0.22.0:http/httpproxy/proxy.go;l=89 — `http_proxy`, `https_proxy`, and `no_proxy`. A more reliable way of logging the relevant info would be to call `FromEnvironment()` in `golang.org/x/net/http/httpproxy` and dump the `HTTPProxy` etc. fields in the returned `Config` struct.